lib/tasks/seed_from_csv.rake

```diff
@@ -30,6 +31,7 @@ namespace :seed_from_csv do
       Merchant.create!(row)
     end
     puts("Merchants imported")
+    ActiveRecord::Base.connection.reset_pk_sequence!('merchants')
```

Comment:
IMPORTANT! this line needs to be at the end of each seeding in order to ensure that the pkeys are reset to allow for item creation and deletion from db